### PR TITLE
User/sheilk/fix gradle

### PR DESCRIPTION
**Pull request by smk2007**
PRs are blocked by failing builds. Remove them to unblock.

1) Remove linux builds
2) Remove azure build
3) Remove failing binary size check build

### CODE REVIEW

**Review by PatriceVignola — APPROVED** [review]

**Comment by snnn** [comment]
Encountered error(s) while parsing pipeline YAML:
/tools/ci_build/github/azure-pipelines/linux-ci-pipeline.yml: (Line: 38, Col: 13, Idx: 888) - (Line: 38, Col: 13, Idx: 888): While scanning a simple key, could not find expected ':'.

File: tools/ci_build/github/azure-pipelines/linux-ci-pipeline.yml [hunk 1]
```diff
@@ -172,3 +173,158 @@ jobs:
       searchFolder: '$(Build.BinariesDirectory)'
       testRunTitle: 'Unit Test Run'
     condition: succeededOrFailed()
+=======
```
Comment (snnn) [inline tools/ci_build/github/azure-pipelines/linux-ci-pipeline.yml:176]: Here! Conflict!

**Comment by smk2007** [comment]
> Line: 38, Col: 13, Idx

Looks like a tab issue... de-tabbing

File: tools/ci_build/github/azure-pipelines/linux-ci-pipeline.yml (hunk 1 above)
Comment (smk2007) [reply tools/ci_build/github/azure-pipelines/linux-ci-pipeline.yml:176]: Ah, looks like i resolved in vscode, but forgot to save before i staged...

**Comment by snnn** [comment]
/tools/ci_build/github/azure-pipelines/linux-ci-pipeline.yml (Line: 170, Col: 3): Unexpected value 'stage'
/tools/ci_build/github/azure-pipelines/linux-ci-pipeline.yml (Line: 182, Col: 3): Unexpected value 'stage'

File: tools/ci_build/github/azure-pipelines/linux-ci-pipeline.yml [hunk 2]
```diff
@@ -172,3 +165,25 @@ jobs:
       searchFolder: '$(Build.BinariesDirectory)'
       testRunTitle: 'Unit Test Run'
     condition: succeededOrFailed()
+
+
+- stage: arm64_build
```
Comment (snnn) [inline tools/ci_build/github/azure-pipelines/linux-ci-pipeline.yml:170]: You can remove this part if it is difficult to resolve.

Comment (snnn) [reply tools/ci_build/github/azure-pipelines/linux-ci-pipeline.yml:170]: The arm64_build part and arm64_test part.

**Review by PatriceVignola — DISMISSED** [review]

**Review by snnn — APPROVED** [review]

**Review by martinb35 — DISMISSED** [review]

**Review by PatriceVignola — APPROVED** [review]

**Review by martinb35 — APPROVED** [review]

**Review by pranavsharma — APPROVED** [review]

**Review by snnn — APPROVED** [review]

**Pull request merged**